src/hotspot/cpu/x86/assembler_x86.cpp

```diff
@@ -11047,6 +11054,7 @@ void Assembler::vbroadcastf128(XMMRegister dst, Address src, int vector_len) {
 
 void Assembler::evbroadcastf64x2(XMMRegister dst, Address src, int vector_len) {
   assert(VM_Version::supports_avx512dq(), "");
+  assert(vector_len == AVX_256bit || VM_Version::supports_avx512vl(), "");
```

Comment:
The assert should be: assert(vector_len == AVX_512bit || VM_Version::supports_avx512vl(), "");